B 26 3 disable thymeleaf logs at startup

nomarchia opened the pull request.

nomarchia committed:
[b-26-3] Disable Thymeleaf logs at startup

Added changes into logback.xml to disable ThymeleafEngine logs;

Brutt commented:
Why did you change pom.xml?

nomarchia commented:
> Why did you change pom.xml?

I accidentally pushed new changes in pom.xml from the second branch to this branch. I can remove it from here if needed. But anyway same changes would be added after second branch merged into develop,

Miari commented:
I understund, that jetty part is from other PR, but as it is here I would like to ask why version 9.4.26 is used? Version 9.4.27 is already available

Brutt commented:
> > Why did you change pom.xml?
> 
> I accidentally pushed new changes in pom.xml from the second branch to this branch. I can remove it from here if needed. But anyway same changes would be added after second branch merged into develop,

Push changes related to branch goal only.

Rooman commented on src/main/resources/logback.xml:
```diff
@@ -27,4 +27,7 @@
    <!-- TODO: Add console appender    -->
+
+    <!-- Disable ThymeleafEngine logs -->
+   <logger name="org.thymeleaf.TemplateEngine" level="OFF"/>
```
Зачем выключать полностью логи тимлифа?

nomarchia replied:
Исправлено на уровень INFO

nomarchia committed:
[b-26-3] Disable Thymeleaf logs at startup

Сhanged logging level for thymeleaf instead of full swtiching off logging;

nomarchia committed:
[b-26-3] Disable Thymeleaf logs at startup

Added changes into logback.xml to not fully disable thymeleaf logs, but change logging level to INFO, in that way thymeleaf doesn't write its debug logs to file;

nomarchia commented:
> Push changes related to branch goal only.

Sure, of course. That's was my mistake. But since those jetty pom.xml changes already merged to develop branch, do we need to remove it from tcurrenthis branch now? Anyway, it's the same as in the develop branch.

nomarchia commented:
> I understund, that jetty part is from other PR, but as it is here I would like to ask why version 9.4.26 is used? Version 9.4.27 is already available

By some reason 9.4.27 wasn't available in my Idea, Maven was not finding it. That's why I tried to use 9.2.46

nomarchia committed:
Update pom.xml

nomarchia committed:
Merge branch 'develop' into b-26-3-disable-thymeleaf-logs-at-startup
